plugins/widget/plugin.js

```diff
@@ -3530,22 +3534,73 @@
 	}
 
 	function setupMask( widget ) {
-		if ( !widget.mask )
+		if ( !widget.mask || !widget.wrapper ) {
```

Comment:
The `setupMask()` function for complete mask (`widget.mask == true`) should be called only once on widget initialization as before. Now, since you merged the logic for handling all mask types into this one method, `setupMask()` for complete mask is called also on editor changes which doesn't make sense. So there should be 2 functions - `setupMask()`/`setupFull/CompleteMask()` which is only called once, on widget init for complete mask and `setupPartialMask()` which is called on widget init and then on widget resize.